server/server_test.go

```diff
@@ -759,6 +760,7 @@ func (s *ServerTestSuite) Test_GetServicesFromEnvVars_ReturnsServices() {
 	os.Setenv("DFP_SERVICE_DISTRIBUTE", strconv.FormatBool(service.Distribute))
 	os.Setenv("DFP_SERVICE_HTTPS_ONLY", strconv.FormatBool(service.HttpsOnly))
 	os.Setenv("DFP_SERVICE_HTTPS_PORT", strconv.Itoa(service.HttpsPort))
+	os.Setenv("DFP_SERVICE_DEFAULT_BACKEND", strconv.FormatBool(service.IsDefaultBackend))
```

Comment:
Why not call it `DFP_SERVICE_IS_DEFAULT_BACKEND`? That would make it consistent and easy to remember that env. var. names are the same as request params with the `DFP_SERVICE_` prefix.